index.html

```diff
@@ -0,0 +1,225 @@
+                </h4>
+                <button class="strong"><a>Download FREE!</a></button>
+            </article>
+            
+            <article class="header__article cell">
+                <img src="./images/screen.png" alt="Screen">
+            </article>
+        </div>
+
+    </header>
+
+
+    <section class="features row" id="Features">
+
+        <div class="features__container container info">
+            <h2 class="features__article">FEATURES</h2>
+        </div>
+
+        <div class="features__container container">
+
+            <article class="features__article cell info">
+                <div class="circle">
+                    <img src="./images/responsive.svg" alt="Mobile device">
+                </div>
+                <p class="strong">FULLY RESPONSIVE</p>
+                <p>
+                    Looks amazing on any device:<br>
+                    smartphone, tablet, laptop and <br>
+                    desktop.
+                </p>
+            </article>
+
+            <article class="features__article cell info">
+                <div class="circle">
+                    <img src="./images/customizable.svg" alt="Tools">
+                </div>
+                <p class="strong">CUSTOMIZABLE</p>
+                <p>
+                    Change the colors, pictures or<br>
+                    any of the selections to suit your<br>
+                    needs.
+                </p>
+            </article>
+
+            <article class="features__article cell info">
+                <div class="circle">
+                    <img src="./images/design.svg" alt="Nice badge">
+                </div>
+                <p class="strong">SLICK AND BEATIFUL DESIGN</p>
+                <p>
+                    Trendy and fresh design, fits<br>
+                    website.
+                </p>
+            </article>
+            
+        </div>
+
+    </section>
+
+
+    <footer class="subfooter footer row ">
+       <img class="subfooter__img" src="./images/logos.png" alt="Companies logos">
+    </footer>
+
+
+    <section class="description row">
+       <img class="description__img cell" src="./images/imac.png" alt="IMac">
+       <article class="description__article cell">
+            <h2>SIMPLE AND BEAUTIFUL</h2>
+            <p>
+                Use Nova theme for your next web project. It is completely
+                customizable so you can change any of the sections to fit your
+                needs. Nova Theme is Free for any kind of use, personal and
+                commercial. Have fun and good luck!
+            </p>
+       </article>
+    </section>
+
+
+    <section class="pricing row" id="Pricing">
+
+        <div class="pricing__container container info">
+            <h2>CHOOSE YOUR PRICING PLAN</h2>
+            <h4>Pick any of our super affordable pricing plans</h4>
+        </div>
+
+        <div class="pricing__container container">
+
+            <article class="pricing__article cell info">
```

Comment:
Tutaj też można zrobić `ul > li` bo to lista pakietów, ale tutaj nie jest tak bardzo istotne. Ta forma, która jest też jest dobra.